action/mount.go

```diff
@@ -31,7 +33,11 @@ func (s *Action) MountsPrint(c *cli.Context) error {
 		return nil
 	}
 	root := simple.New(color.GreenString(fmt.Sprintf("gopass (%s)", s.Store.Path())))
-	for alias, path := range s.Store.Mounts() {
+	mounts := s.Store.Mounts()
+	mps := s.Store.MountPoints()
+	sort.Sort(store.ByPathLen(mps))
+	for _, alias := range mps {
+		path := mounts[alias]
```

Comment:
this returns "" for the path, if alias isn't found within mounts. Shouldn't happen (tm), as s.Store.Mounts() and .MountPoints() use the same underlying map. Maybe anyways because of the (tm)?